docs/cmd/kn.md

```diff
@@ -16,11 +16,13 @@ Eventing: Manage event subscriptions and channels. Connect up event sources.
       --config string       config file (default is $HOME/.kn/config.yaml)
```

Comment:
just to make it clear what the diff is between this and the --kubeconfig value, can you make this `kn config file...`  - just add `kn` to the description

Reply:
it’s obvious from the last part `(default ...)` but does not hurt to add